### ISSUE TITLE
Integrate Song + Score with Vault

### ISSUE THREAD

**Issue by rosibaj** [icgc-argo/platform-ui#382]
after discussion with @andricDu we do want to integrate vault with song and score, it should be an straight forward task based on how easy it was to integrate with ego

- [x] 1- add dependency to each project
- [ ] 2- add proper configuration in helm
- [ ] 3- add the secrets for vault in Dev and QA
- [ ] 4- delete and reinstall the charts to confirm they pickup things correctly in Dev & QA

There is no code required for this to be implemented.

**Comment by andricDu** [comment]
There is a problem regarding how we initialize the databases with helm. Currently blocked by this for the Song chart.

**Comment by andricDu** [comment]
I'm marking this as blocked until we can resolve the issue with bootstrapping a database with credentials.

**Comment by rosibaj** [comment]
Future work:
- @henro001 to help provide plan for vault to provision postgres credentials (short-lived)
